layouts/event/single.html

```diff
@@ -30,6 +30,11 @@
             {{ with .Params.duration }}<div>
                 <strong>{{ T "event-duration" }}</strong><br> {{ . }}
             </div>{{ end }}
+            <br>{{ T "blog-licensed-under" }} <a
+                    href="{{ T (printf "lic-url-%s" ($.Scratch.Get "license")) }}">{{ T (printf "lic-%s" ($.Scratch.Get "license")) }}</a>
+                <span class="hidden" xmlns:dct="http://purl.org/dc/terms/" href="http://purl.org/dc/dcmitype/Dataset"
+                    property="dct:title" rel="dct:type">{{ .Title }}</span>
+                    {{ with .Params.notices }}<p>{{ . | markdownify }}</p>{{ end }}
```

Comment:
The notices won't fit in the small space under the video. Take a look at the build preview: https://deploy-preview-410--datenanfragen.netlify.app/en/verein/event/okl-berlin-05-2020/

Maybe move those someplace else.

Reply:
Any suggestion where should I move them?

Reply:
Maybe just to the bottom of the page? That seems like an appropriate place for notices.